<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileXyzReaderDialog</class>
 <widget class="QDialog" name="FileXyzReaderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>228</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>476</width>
    <height>228</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>476</width>
    <height>228</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>File XYZ converter</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="hlFile">
     <item>
      <widget class="QLabel" name="lFile">
       <property name="text">
        <string>File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leFile"/>
     </item>
     <item>
      <widget class="QPushButton" name="pbFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="hlDelimitor">
     <item>
      <widget class="QLabel" name="lDelimitor">
       <property name="text">
        <string>Delimitor caracter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbDelimitor"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="hlTable">
     <item>
      <widget class="QLabel" name="lTable">
       <property name="text">
        <string>Table</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbTable"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="hlCalculate">
     <item>
      <spacer name="hsCalculateLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbImport">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hlCalculateRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lOperationName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="maximumSize">
      <size>
       <width>468</width>
       <height>25</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FileXyzReaderDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FileXyzReaderDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbFile</sender>
   <signal>clicked()</signal>
   <receiver>FileXyzReaderDialog</receiver>
   <slot>selectFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbImport</sender>
   <signal>clicked()</signal>
   <receiver>FileXyzReaderDialog</receiver>
   <slot>import()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>import()</slot>
  <slot>selectFile()</slot>
  <slot>typeChanged(QString)</slot>
 </slots>
</ui>
